lib/loader.js

```diff
@@ -67,6 +71,13 @@ function readSpecFile(file, options) {
     }
 }
 
+const readGlob = async (pattern, options = {}) => {
```

Comment:
I did not want to change `readOrError` to avoid breaking other functionality. That's why I wrapped it in another method.